catalog/ui/catalog-ui-search/src/main/webapp/component/navigation-right/navigation-right.less

```diff
@@ -16,6 +16,13 @@
         width: auto;
         overflow: hidden;
         text-overflow: ellipsis;
+        margin-right: 9rem;
+    }
+
+    .navigation-item.navigation-icon {
+        margin-right: 0.5em;
+        padding-left: 0.575em;
```

Comment:
This is not going to be a good idea.  The border doesn't help out, and the margin creates a deadzone for clicks.  In addition, the padding will make it so the icon isn't centered.